py-utils/src/utils/msg_bus/message_bus/template/singleton.py

```diff
@@ -0,0 +1,7 @@
+
```

Comment:
Move the template to outside of message_bus and separate patch. 
Best folder is py-utils/src/template/

Reply:
Moved